portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,47 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ani's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+
```

Comment:
Nit: usually more than one blank line in a row in code is discouraged as too much white space.